data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,92 @@ mission "FW Epilogue: Danforth"
+			`	As his tangent about his new invention continues, you think to yourself about the old times, the war, and all the people you've met along the way.`
+			`	"How about we get out of here <first>? My work is done for today, an I know a homely little bar that's only a short walk."`
+			choice
+				`	"Of course. It's not like there's a war to worry about anymore."`
+			`"Ha! Well of course, the war kept all of us plenty busy." He leads you out of the facility while telling an anecdote that must be humor for an engineer.`
+			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
+			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
+			`	"No way you're from New Boston too! What are the chances?" you respond.`
+			`	He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`
+				decline
+		event "plasma repeater advanced" 95
+		event "plasma repeater basics" 155
+	npc disable save
+		government "Test Dummy"
+		personality disables heroic staying
+		ship "Leviathan (Plasma Repeater)" `Hunk`
+	on complete
+		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon. Increased fire rate, even at lower damage per hit, comes out with better overall results.`
+		conversation
+			scene "scene/plasma_scene"
+			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with a large gun. It resembles an old Plasma Cannon, but it is somewhat different. Presumably, this is the weapon that you were just attacked with in orbit. "It runs much hotter, you see," he explains. "But the results are impressive."`
+			choice
+				`	"What was that?"`
+					goto what
+				`	"It looks impressive, but it didn't do too much damage."`
+					goto damage
+			label what
+			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That was the Plasma Repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
```

Comment:
```suggestion
			`	"Heh, it's been a while since we've seen a good upgrade to the cannon," he says. "That was the plasma repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
```